### DISCUSSION TITLE
[Dependabot alerts] number of items saved

### DISCUSSION THREAD

**Discussion by GitSumito** [community/community#40589] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a question about "Dependabot alerts".
May I ask what is the maximum number of items saved detected by Dependabot alerts?
I am concerned that old alerts will be lost.

**Comment by ossanna16** [comment]
Hi there @GitSumito and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
